### ISSUE TITLE
ansible-lint fails to load role lookups for templates when used in block

### ISSUE THREAD

**Issue by Atoms** [ansible-community/ansible-lint#1489]
Exact same issue is reported earlier, but it seems abandoned, as all new comments are ignored (https://github.com/ansible-community/ansible-lint/issues/375)

##### Summary

When there is template lookup in the block then ansible lint shows warning it cannot find a template.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
# ansible --version
ansible 2.10.6
  config file = /home/xxx/projects/ansible/ansible.cfg
  configured module search path = ['/home/xxx/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/xxxx/projects/ansible/.env/lib/python3.8/site-packages/ansible
  executable location = /home/xxxx/projects/ansible/.env/bin/ansible
  python version = 3.8.5 (default, Jan 27 2021, 15:41:15) [GCC 9.3.0]

# ansible-lint --version
ansible-lint 5.0.2 using ansible 2.10.6
```

- ansible installation method: one of source, pip, OS package
pip
- ansible-lint installation method: one of source, pip, OS package
pip

##### STEPS TO REPRODUCE

Create 2 roles with 1 task:
working task:
```yaml
- name: "Show custom role template"
  debug:
    msg: "{{ lookup('template', 'template_test2.yml.j2') }}"
```
not working task:
```yaml
- block:
    - name: "Show custom role template"
      debug:
        msg: "{{ lookup('template', 'template_test1.yml.j2') }}"
```
##### Desired Behaviour

ansible-lint does not throw a warning for tasks just being in block

##### Actual Behaviour

```console
$ ansible-lint -p test.yml -vvv
DEBUG    Logging initialized to level 10
DEBUG    Options: Namespace(colored=True, config_file=None, cwd=PosixPath('/home/xxxx/projects/ansible'), display_relative_path=True, exclude_paths=['/home/xxxx/projects/ansible/.yamllint', '/home/xxxx/projects/ansible/collections', '/home/xxxx/projects/ansible/.env', '/home/xxxx/projects/ansible/config-connector-init', '/home/xxxx/projects/ansible/cert-manager.yaml', '/home/xxxx/projects/ansible/requirements.yaml', '/home/xxxx/projects/ansible/.gitlab-ci.yml'], extra_vars=None, format='rich', kinds=[{'requirements': 'requirements.yml'}, {'requirements': '**/meta/requirements.yml'}, {'reno': 'releasenotes/*/*.{yaml,yml}'}, {'playbook': '**/playbooks/*.{yml,yaml}'}, {'playbook': '**/*playbook*.{yml,yaml}'}, {'role': '**/roles/*/'}, {'tasks': '**/tasks/*.{yaml,yml}'}, {'handlers': '**/handlers/*.{yaml,yml}'}, {'vars': '**/{host_vars,group_vars,vars,defaults}/**/*.{yaml,yml}'}, {'meta': '**/meta/main.{yaml,yml}'}, {'yaml': '.config/molecule/config.{yaml,yml}'}, {'requirements': '**/molecule/*/collections.{yaml,yml}'}, {'yaml': '**/molecule/*/{base,molecule}.{yaml,yml}'}, {'playbook': '**/molecule/*/*.{yaml,yml}'}, {'yaml': '**/*.{yaml,yml}'}, {'yaml': '**/.*.{yaml,yml}'}], lintables=['test.yml'], listrules=False, listtags=False, loop_var_prefix=None, mock_modules=[], mock_roles=[], offline=False, parseable=True, parseable_severity=False, progressive=False, quiet=False, rulesdir=[], rulesdirs=['/home/xxxx/projects/ansible/.env/lib/python3.8/site-packages/ansiblelint/rules'], skip_action_validation=True, skip_list=['204', 'risky-shell-pipe', 'meta-no-info'], tags=[], use_default_rules=False, verbosity=3, version=False, warn_list=['experimental'])
Added ANSIBLE_ROLES_PATH=~/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:roles
DEBUG    Loading rules from /home/xxxx/projects/ansible/.env/lib/python3.8/site-packages/ansiblelint/rules
Loading custom .yamllint config file, this extends our internal yamllint config.
WARNING  Overriding detected file kind 'yaml' with 'playbook' for given positional argument: test.yml
INFO     Discovering files to lint: git ls-files -z *.yaml *.yml
INFO     Executing syntax check on test.yml (0.30s)
[WARNING]: Unable to find 'template_test1.yml.j2' in expected paths (use -vvvvv to see paths)
DEBUG    Examining roles/test_role1/tasks/main.yml of type tasks
DEBUG    Examining roles/test_role2/tasks/main.yml of type tasks
DEBUG    Examining test.yml of type playbook
```

**Comment by juanvalino** [comment]
Same problem here
